docker/README.md

```diff
@@ -27,19 +48,49 @@ mkdir -p data/server
 ```
 
 4. Start the service
- 
+
 ```bash 
 USER_ID=$(id -u) GROUP_ID=$(id -g) docker-compose up -d
 ```
 
-# AWS EC2 Setup
+Congratulations! 
+Check to see if everything is working by visiting `HOSTNAME/ping`
+
+# Instance Admin Tasks
+
+## Add a new user identity
+
+To add a new user identity, run the following on the host machine:
+
+`docker run --rm -it --network docker_db -e NEO4J_URL=bolt://neo4j:7687 -e NEO4J_USER=<USER> -e NEO4J_PASS=<PASSWORD> <ALCHEMISCALE_DOCKER_IMAGE> identity add -t user -i <user identity> -k <user key>`
+
+The important bits here are:
+1. `--network docker_db`
+We need to make sure the docker container we are using can talk to the database container.
 
-The following series of commands are sufficient for setting up Docker on an AWS
-EC2 instance running Amazon Linux.
+2. `-e NEO4J_URL=bolt://neo4j:7687 -e NEO4J_USER=<USER> -e NEO4J_PASS=<PASSWORD>`
+We need to pass in these environment variables so that the container can talk to the database. 
+These should match the values set in `.env`.
+
+
+## Add a new compute identity
+
+To add a new compute identity, perform the same operation as for user identities given above, **but replace `-t user` with `-t compute`**.
+Compute identities are needed by compute services to authenticate with and use the compute API.
+
+# Annex
+
+The stuff below is not needed for deployment but useful to reference.
+
+## AWS EC2 Setup
+
+The following series of commands are sufficient for setting up Docker on an AWS EC2 instance running Amazon Linux. 
+Updated for Amazon Linux 2023.
+However, please use the launch `template alchemiscale-template` to spin up a new instance as it will use an AMI that has all dependences installed (so you will not have to do the steps below).
```

Comment:
From my comment above: these are not unnecessary, they are crucial. These should be our canonical instructions for deploying `alchemiscale` to an AWS environment, ideally with no assumptions on pre-existing resources defined.